lib/Models/Leaflet.js

```diff
@@ -201,6 +201,28 @@ Leaflet.prototype.computePositionOnScreen = function(position, result) {
     return result;
 };
 
+/**
+ * Add and remove attribution text.
+ * @param {attribution} the attribution object of each Catalog item.
+ */
+Leaflet.prototype.addAttribution = function(attribution){
+    this.map.attributionControl.addAttribution(leafletCredit(attribution));
+};
+
+Leaflet.prototype.removeAttribution = function(attribution){
+    this.map.attributionControl.removeAttribution(leafletCredit(attribution));
+};
+
+/**
+ * A convenient function for handling leaflet credit display
+ * @param {attribution object} for leaflet to display as text or link
```

Comment:
Renaming `attObj` to `credit` would be a nice improvement, too.  We try to avoid abbreviations in the public API.